examples/cartpole.py

```diff
@@ -101,9 +105,9 @@ def __init__(self, env):
         ngoal = EvaluationLink(PredicateNode("Reward"), NumberNode("0"))
 
         # Call super ctor
-        OpencogAgent.__init__(self, env, action_space, pgoal, ngoal)
+        super().__init__(env, action_space, pgoal, ngoal)
 
-    def plan(self, goal, expiry):
+    def plan(self, goal, expiry) -> List:
```

Comment:
These are type hints - they help with development because you can have a linter point at inconsistent types and also allows your IDE to show you documentation and property hints if it knows what type that particular value is.